backend/index.js

```diff
@@ -6,12 +6,26 @@ app.use(express.json())
 
 const port = 8080;
 
-app.get('/',(req,res)=>{
-    res.send('Hello world');
+const zipcode = 75080;
+const distance = 50;
+
+app.get('/api/zipcodes',(req,res)=>{
+
+    const headers = {headers: {authorization: process.env.API_KEY}}
```

Comment:
change the name of the API key to something that describes what it is since there will be multiple api keys

Reply:
does this API actually require a key? It doesnt seem like it does